docscan/session/retrieve/check_response.go

```diff
@@ -6,14 +6,14 @@ import (
 
 // CheckResponse represents the attributes of a check, for any given session
 type CheckResponse struct {
-	ID             string           `json:"id"`
-	Type           string           `json:"type"`
-	State          string           `json:"state"`
-	ResourcesUsed  []string         `json:"resources_used"`
-	GeneratedMedia []GeneratedMedia `json:"generated_media"`
-	Report         ReportResponse   `json:"report"`
-	Created        *time.Time       `json:"created"`
-	LastUpdated    *time.Time       `json:"last_updated"`
+	ID             string            `json:"id"`
+	Type           string            `json:"type"`
+	State          string            `json:"state"`
+	ResourcesUsed  []string          `json:"resources_used"`
+	GeneratedMedia []*GeneratedMedia `json:"generated_media"`
+	Report         *ReportResponse   `json:"report"`
+	Created        time.Time         `json:"created"`
+	LastUpdated    time.Time         `json:"last_updated"`
```

Comment:
We might need to add `omitempty` to properties if they're not always present. I believe we have done this in all other SDKs.

Reply:
As discussed `omitempty` isn't necessary when unserializing